### ISSUE TITLE
open-image-url conflicts with application cache

### ISSUE THREAD

**Issue by dyoo** [dyoo/whalesong#63]
open-image-url is not safe when used with application caching in place.  It looks like Chrome will not respect xmlhttprequests when it's in appcache mode.

**Comment by dyoo** [comment]
Ok, I need to add the following to the manifest to let other network resources work out:

```
 # All other resources (e.g. sites) require the user to be online.
 NETWORK:
 *
```

**Comment by dyoo** [comment]
Will be fixed when I release 1.8  (afaea74660679ac7d3af2d5769911c55463c1031)
